packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -310,7 +311,10 @@ export const HelpCenterContactForm = () => {
 			section: sectionName,
 		} );
 
+		const savedCurrentSite = currentSite;
```

Comment:
I'm curious - why is this needed? `resetStore` should put the store in correct, clear state. I'm worried of possible side-effects if we modify it like this.

Reply:
resetStore also unsets the currentSite. It's empty without this.
Try getting a response, then hitting "close" then trying again - the form will be broken.

Reply:
Right, but where did the site come from in the beginning? Feels like that logic is not being triggered and that should be addressed "from the beginning/front", not at the end/cleanup phase. What if the user switches the site in the meantime - will the saved site remain the same in this context?